tests/v1_upgrade_v2/phase2_verify_after_upgrade.py

```
@@ -0,0 +1,282 @@
+
+    print(f"\n✅ Deleting v2 collection '{V2_COLLECTION_NAME}'")
+    client.delete_collection(V2_COLLECTION_NAME)
+    assert not client.has_collection(V2_COLLECTION_NAME), f"v2 collection '{V2_COLLECTION_NAME}' should be gone after delete"
+    print(f"   ✅ v2 collection '{V2_COLLECTION_NAME}' successfully deleted")
+
+    # After delete, get_collection(name) should raise (sdk_collections row for this name is gone)
+    print(f"   Verifying get_collection('{V2_COLLECTION_NAME}') raises after deletion (sdk_collections row removed)")
+    print(f"   Verifying get_collection('{V2_COLLECTION_NAME}') raises ValueError after deletion")
+    try:
+        deleted_coll = client.get_collection(V2_COLLECTION_NAME)
+        assert False, f"get_collection('{V2_COLLECTION_NAME}') should raise ValueError after deletion, but returned {deleted_coll}"
```

Ruff (0.14.14)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)